scenes/src/app/app.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
```

Comment:
```suggestion
  template: '<router-outlet></router-outlet>',
```
Can we inline the template at remove the style file since this is such a small thing?